lib/doorkeeper-jwt.rb

```diff
@@ -18,7 +18,11 @@ def self.token_payload(opts = {})
       Doorkeeper::JWT.configuration.token_payload.call opts
     end
 
-    def self.secret_key
+    def self.secret_key(opts)
```

Comment:
@chriswarren I'm not sure if you want me to change this one or not... I noticed that hound was complaining about single-quotes which are being used often in other files.